.github/workflows/docker-publish.yml

```diff
@@ -0,0 +1,24 @@
+name: Docker Publishing
+
+on:
+  push:
+    branches:
+      - '*'
+    tags:
+      - '[0-9]+\.[0-9]+\.[0-9]+'
+
+jobs:
+  publish:
+
+    runs-on: ubuntu-20.04
```

Comment:
I would think it worthwhile to start with a newer LTS version of Ubuntu (22.04). We shouldn't *start* a project with out-of-date software, right :) If you do, you'll need to upgrade/change the postgresql apt packages.